scss/_base_icon-definitions.scss

```
@@ -172,3 +172,337 @@ $social-icon-size: map-get($icon-sizes, heading-icon--small);
 @mixin vf-icon-pin($color) {
   background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M11.547.119l4.412 4.412-1.06 1.06-.366-.365-2.205 2.204-1.385 2.598 1.128 1.129-1.06 1.06-3.012-3.011-4.523 4.524-1.898.916.744-1.882 4.617-4.618-3.01-3.01 1.06-1.06 1.028 1.027 2.819-1.252 2.16-2.161-.51-.51 1.06-1.061zm.381 2.76L9.897 4.912a1.5 1.5 0 01-.329.248l-.123.062L7.16 6.237l2.678 2.678 1.167-2.19a1.5 1.5 0 01.164-.246l.1-.11 2.075-2.076-1.415-1.414z' fill='#{vf-url-friendly-color($color)}' fill-rule='nonzero'/%3E%3C/svg%3E");
 }
+
+// NEW
```

I guess `// NEW` can quickly get outdated ;) Maybe just call them `// ADDITIONAL`?

```suggestion
// ADDITIONAL
// Generated with icon-svgs-to-mixins script
```

---

Very true